hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/RabbitMqBackendResolver.java

```diff
@@ -28,6 +36,18 @@
     }
 
     Builder entityBuilder = getBackendEntityBuilder(BackendType.RABBIT_MQ, routingKey.get(), event);
-    return Optional.of(entityBuilder.build());
+
+    setAttributeForFirstExistingKey(
```

Comment:
If the attribute was not set in the entity before, let's not add it to the entity